src/Socket.js

```diff
@@ -299,6 +299,7 @@ export default class Socket extends EventEmitter {
      */
     write(buffer, encoding, cb) {
         const self = this;
+        if (encoding && typeof encoding === 'function') cb = encoding;
```

Comment:
JAVASCRIPT: THE GOOD PARTS notes its always better to use `{ }` to provide scope because its less error prone. For example:

```
   if (x)
      a()
```

Will become:
```
if (x)
   a()
   b()
```

Reply:
@jleach, this is part of the prettier/eslint configuration. It avoids {} if it is a one liner.